packages/venia-concept/package.json

```diff
@@ -37,7 +37,8 @@
   },
   "homepage": "https://github.com/magento/pwa-studio/tree/master/packages/venia-concept#readme",
   "dependencies": {
-    "@magento/pwa-buildpack": "~7.0.0"
+    "@magento/pwa-buildpack": "~7.0.0",
+    "@magento/venia-sample-payments-checkmo": "link:/home/roettigl/vscode/pwa-studio/packages/extensions/venia-sample-payments-checkmo"
```

Comment:
You should probably use the version of the package directly instead of using a link. Since the package is part of the same main package (pwa-studio), yarn should be able to link them internally.